virtcontainers/clh.go

```diff
@@ -425,8 +432,62 @@ func (clh *cloudHypervisor) hypervisorConfig() HypervisorConfig {
 }
 
 func (clh *cloudHypervisor) resizeMemory(reqMemMB uint32, memoryBlockSizeMB uint32, probe bool) (uint32, memoryDevice, error) {
-	clh.Logger().WithField("function", "resizeMemory").Warn("not supported")
-	return 0, memoryDevice{}, nil
+	memDev := memoryDevice{probe: probe}
+	if reqMemMB == 0 {
+		return 0, memDev, errors.New("Can not resize memory to 0")
+	}
```

Comment:
nit: it should be covered by the following check on `if currentMem > newMem `.

Reply:
I decided to add this corner case because resize to 0 is meaning of something went really wrong, in the case when memory is just reduced  only a warning is set and no operation is performed but not errors are returned.